internal/receiver/smartagentreceiver/log_test.go

```diff
@@ -83,6 +83,14 @@ func TestRedirectMonitorLogs(t *testing.T) {
 		logToZap.redirect(monitor2LogrusKey, monitor2ZapLogger)
 		logToZap.redirect(monitor3LogrusKey, monitor3ZapLogger)
 
+		expectedLogrusLevel := logrusLevel
+		if logrusLevel == logrus.TraceLevel {
+			expectedLogrusLevel = logrus.DebugLevel
+		}
+		require.Equal(t, monitor2LogrusKey.Level, expectedLogrusLevel)
```

Comment:
Looks like `monitor2LogrusKey.Level, expectedLogrusLevel` args should be swapped, no?